Cinecon/VisitorsMenu.cs

```diff
@@ -12,6 +12,7 @@ public static class VisitorsMenu
     {
         private static List<KeyValuePair<string, Action>> _genres;
         private static KeyValuePair<string, string[]> _dayAndTimes;
+        private static List<Seat> SelectedSeats { get; set; }
```

Comment:
Waar is deze field voor nodig? Het wordt nergens geinitialized.

Reply:
Niet, goed dat je het zegt